mobx/lib/src/api/reaction.dart

```diff
@@ -6,7 +6,7 @@ import 'package:mobx/src/core.dart';
 ///
 /// Optional configuration:
 /// * [name]: debug name for this reaction
-/// * [delay]: debouncing delay in milliseconds
+/// * [delay]: throttling delay in milliseconds
```

Comment:
There are other reactions like autorun that have the same kind of docs. Would be good to sweep through other reaction APIs and replace debouncing with throttling

Reply:
I updated autorun, I will search for others and report back.